gitflow.html

```diff
@@ -52,31 +52,171 @@
 ## A branching model for Git
         
 .right[![Right-aligned image](https://avatars2.githubusercontent.com/u/52010894?s=200&v=4)]
-        
+
 ---
-               
-## The Branches
+
+## What Is GitFlow??
+
+--
+GitFlow is a branching model for Git, created by Vincent Driessen. It has attracted a lot of attention because it is very well suited to collaboration and scaling the development team. We think this model is very well suited with our current release process at ESDC.  
```

Comment:
Some wording/phrasing/grammar suggestions.
```suggestion
GitFlow is a branching model for Git, created by Vincent Driessen.
It has attracted a lot of attention because it is well suited for collaboration and scaling projects.
We have successful experiences using GitFlow cohesively with our existing release process at ESDC.  
```